app/components/Views/MultichainTransactionsView/MultichainTransactionsView.tsx

```diff
@@ -30,31 +30,30 @@ const MultichainTransactionsView = () => {
     selectSelectedInternalAccountFormattedAddress,
   );
   const [loading, setLoading] = useState(true);
-  const [transactions, setTransactions] = useState<Transaction[]>([]);
   const solanaAccountTransactions = useSelector(
     selectSolanaAccountTransactions,
   );
 
-  useEffect(() => {
-    setLoading(true);
+  const transactions = useMemo(() => {
+    if (
+      solanaAccountTransactions &&
+      'transactions' in solanaAccountTransactions
+    ) {
+      return solanaAccountTransactions.transactions;
+    }
+    return [];
+  }, [solanaAccountTransactions]);
```

Comment:
Question around this `useMemo`. Is it not simpler to do the following?

```suggestion
const transactions = useMemo(() => 
  solanaAccountTransactions?.transactions || [], 
  [solanaAccountTransactions]
);
```